ci/test/00_setup_env_native_msan.sh

```diff
@@ -16,7 +16,7 @@ export CONTAINER_NAME="ci_native_msan"
 export PACKAGES="ninja-build"
 # BDB generates false-positives and will be removed in future
 export DEP_OPTS="DEBUG=1 NO_BDB=1 NO_QT=1 CC=clang CXX=clang++ CFLAGS='${MSAN_FLAGS}' CXXFLAGS='${MSAN_AND_LIBCXX_FLAGS}'"
-export GOAL="install"
+export GOAL="all"
```

Comment:
Why does this need to change? The regular msan job is not broken afaik.

Reply:
It does seem more correct. Perhaps in the future we can even warn users when they're about to install a debug, sanitizer or otherwise potentially very slow binary.

Reply:
> Why does this need to change? The regular msan job is not broken afaik.

Right. It wasn't my intention. Reverted.